pkg/exasol-rest-api/properties_file_parser.go

```diff
@@ -0,0 +1,45 @@
+package exasol_rest_api
+
+import (
+	"errors"
+	"gopkg.in/yaml.v3"
+	"os"
+)
+
+func getPropertiesFromFile(filepath string, properties *ApplicationProperties) error {
+	propertiesFile, err := openFile(filepath)
+	if err != nil {
+		return err
+	}
+	err = decodePropertiesFile(propertiesFile, properties)
+	if err != nil {
+		return err
+	}
+	closeFile(propertiesFile)
+	return nil
+}
+
+func openFile(filepath string) (*os.File, error) {
+	file, err := os.Open(filepath)
+	if err != nil {
+		return nil, err
+	} else if file == nil {
+		return nil, errors.New("properties file doesn't exist")
```

Comment:
Output expected filepath, maybe use error code

Reply:
I'll improve error messages and address this finding when I work on errors ticket: https://github.com/exasol/exasol-rest-api/issues/10